Calculate_mean_activation_per_filter_in_specific_layer_given_an_image.ipynb

```diff
@@ -9,42 +9,46 @@
   },
```

Comment:
<p>Why did you remove the <code>activations.close()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fg91/visualizing-cnn-feature-maps/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>